docs/api/materials/ShaderMaterial.html

```diff
@@ -31,11 +31,11 @@
 		var material = new THREE.ShaderMaterial( {
 
 			uniforms: {
-				time: { type: "f", value: 1.0 },
```

Comment:
Actually, maybe they still need to be an object because of the `needsUpdate = true`. So maybe...

```js
uniforms: {
    time: new THREE.Uniform( 1.0 ),
    resolution: new THREE.Uniform( new THREE.Vector2() )
}
```